unit/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ find_package(OpenCV REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(Pangolin 0.4 REQUIRED)
 find_package(spdlog REQUIRED)
-include_directories(${OpenCV_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIR} ${Pangolin_INCLUDE_DIRS} src/main/include)
-add_executable(${PROJECT_NAME} src/main/cpp/robot.cpp src/main/include/robot/state.h src/main/include/robot/outputs.h src/main/cpp/subsystems/vision.cpp src/main/include/subsystems/vision.h src/main/include/subsystems/subsystemBase.h src/main/cpp/robot/commands.cpp src/main/include/robot/commands.h src/main/cpp/subsystems/display.cpp src/main/include/subsystems/display.h src/main/include/config/constants.h src/main/cpp/utils/utils.cpp src/main/include/utils/utils.h src/main/cpp/robot/control.cpp src/main/include/robot/control.h src/main/include/subsystems/general.h src/main/cpp/subsystems/general.cpp src/main/cpp/robot/state.cpp src/main/cpp/robot/outputs.cpp)
-target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBS} ${EIGEN3_LIBS} ${Pangolin_LIBRARIES} spdlog::spdlog)
+find_package(CURL REQUIRED)
+include_directories(${OpenCV_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIR} ${Pangolin_INCLUDE_DIRS} src/main/include restclient-cpp/include)
```

Comment:
Add linux / unix install instructions in RUN.md

Reply:
There shouldn't be any extra instructions for linux except for installing curl, the restclient-cpp is already bundled

Reply:
curl needs to be installed on linux. I want a singular list of commands so its very easy to get up and running on a brand new machine. Should only be one apt command or brew for mac